gui/src/app/Scripting/webR/useWebR.ts

```diff
@@ -162,6 +162,7 @@ const outputLoop = async (
           canvas.setAttribute("width", "1008");
           canvas.setAttribute("height", "1008");
           canvas.style.width = "100%";
+          canvas.style.backgroundColor = "white";
```

Comment:
is it desired that the plots always have a white background even in dark theme?

Reply:
Yes, because the plotting engines in e.g. matplotlib don't know they're in a dark theme, so will use black text, etc.